### ISSUE TITLE
Native resolution results in VHS-like horizontal lines on screen [Report]

### ISSUE THREAD

**Issue by ACCESS-DENIIED** [holoiso-eol/holoiso#676]
## Your Issue:
```
Choosing Native resolution in any game displays VHS-like horizontal lines on screen constantly. Tested on multiple monitors. My primary monitor is 3440x1440 (21:9) and my other monitors are 1920x1080, all of them have the same results. Strangely enough, lowering the resolution (on any of the monitors) eliminates this issue entirely. The Issue is also not present while in desktop mode.
```

## Your device's full specs
```
CPU: Intel i7-11700K
GPU: Nvidia RTX 3070
```

## Your OS version (`cat /etc/os-release | grep VARIANT_ID | cut -d '"' -f 2`):
```
ISO Update 3.3 (Bootstrap based on snapshot20221101.1327)
```

## (Only for SteamOS Session loading issues) Have you tried launching `gamescope-session` from terminal? If yes, provide a log.
```
N/A
```

## Your installation type:
```
deckperience
```

## Have you updated your installation recently?
```
nawww
```

**Comment by ACCESS-DENIIED** [comment]
Update: Had to reinstall 3.3 last night and now these lines are displaying on any resolution I choose. Is it possible that it's a driver issue if this didn't happen on 2560x1080 last night but following a reinstall of the OS, the lines are now on any resolution? If so, how do I fix this?

**Comment by ACCESS-DENIIED** [comment]
Update 2: I've fixed the issue.

This bug is specific to the "Native" resolution option in the game properties. If you choose "Native" (Which I did because my main desktop has an ultrawide display which there isn't a resolution option for) then choosing the same resolution in-game will result in this issue, at least on my end anyway. This leaves me with no other option than to force my display to a 16:9 aspect ratio.

Fix: Select 4K as the resolution option in the game properties instead of "Native".
